### PR TITLE
ledger-live-desktop: 2.16.0 -> 2.17.1

### PR DESCRIPTION
**Pull request by Th0rgal**
###### Motivation for this change
The application displayed a message suggesting to update the software. I also added myself as a maintainer to receive notifications about further updates and help if I can.

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/blockchains/ledger-live-desktop/default.nix [hunk 1]
```diff
@@ -30,8 +30,7 @@ in appimageTools.wrapType2 rec {
     description = "Wallet app for Ledger Nano S and Ledger Blue";
     homepage = "https://www.ledger.com/live";
     license = licenses.mit;
-    maintainers = with maintainers; [ thedavidmeister nyanloutre RaghavSood ];
+    maintainers = with maintainers; [ thedavidmeister nyanloutre RaghavSood th0rgal ];
     platforms = [ "x86_64-linux" ];
   };
-}
-
+}
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/ledger-live-desktop/default.nix:36]: Please add the missing final new line back.

**Comment by RaghavSood** [comment]
Result of `nixpkgs-review pr 105130` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ledger-live-desktop</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105130` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ledger-live-desktop</li>
  </ul>
</details>

**Comment by prusnak** [comment]
ccing all maintainers:  @thedavidmeister @nyanloutre @RaghavSood @th0rgal

Honest question - what's the point of maintaining this in nixpkgs? AppImage should work just fine via appimage-run and with the release cycle of ~2 weeks this will get obsoleted by the time it is merged to master, not to mention reaching the unstable channel.

**Comment by Th0rgal** [comment]
To be honest I didn't know appimage-run. However, ledger-live-desktop is a rather sensitive piece of software and with all the phishing attempts going on, I think it's more important to be sure to have an official build than to have the latest version.

**Comment by RaghavSood** [comment]
Could you please squash the formatting and update commits into a single one, and split adding yourself to the maintainer list into a separate commit so that we can safely revert the update without kicking you off the list if necessary?

Not that I believe we'll need to revert, just a safety practice

LGTM apart from that

Agreed that there is some security value in having ledger-live as a maintained package - the r-ryan or one of us usually picks it up within a few hours of a new release, but I could set up some automation for it if we start slipping.

**Comment by Th0rgal** [comment]
> Could you please squash the formatting and update commits into a single one, and split adding yourself to the maintainer list into a separate commit so that we can safely revert the update without kicking you off the list if necessary?
> 
> Not that I believe we'll need to revert, just a safety practice
> 
> LGTM apart from that
> 
> Agreed that there is some security value in having ledger-live as a maintained package - the r-ryan or one of us usually picks it up within a few hours of a new release, but I could set up some automation for it if we start slipping.

Here we go!

**Review by prusnak — CHANGES_REQUESTED** [review]
The commit messages should read:
* `ledger-live-desktop: 2.16.0 -> 2.17.1` (note the extra `:` after the word `desktop`)
* `ledger-live-desktop: add th0rgal as maintainer`

**Comment by nyanloutre** [comment]
> ccing all maintainers: @thedavidmeister @nyanloutre @RaghavSood @Th0rgal
> 
> Honest question - what's the point of maintaining this in nixpkgs? AppImage should work just fine via appimage-run and with the release cycle of ~2 weeks this will get obsoleted by the time it is merged to master, not to mention reaching the unstable channel.

I run my system on unstable and it's much more convenient to get software updates from the nix store than to manually download an AppImage and store it at a random location.

**Pull request merged**